code/modules/mob/new_player/new_player.dm

```diff
@@ -246,7 +246,12 @@
 	if(!job)
 		return 0
 	if((job.current_positions >= job.total_positions) && job.total_positions != -1)
-		return 0
+		if(job.title == "Assistant")
+			for(var/datum/job/J in SSjob.occupations)
+				if(J && J.current_positions < J.total_positions && J.title != job.title)
+					return 0
+		else
+			return 0

```

Comment:
Why's it gotta check twice? Because if I passed the "get around full slots" as a var, it could be href spoofed